docs/nodes/configuration.md

```diff
@@ -202,35 +220,59 @@ pprof-laddr = ""
 #######################################################
 [p2p]
 
+# Enable the new p2p layer.
```

Comment:
Can we link to the newly added docs on the new p2p layer?

Reply:
Nope. I can't even find docs on the old p2p layer other than in the spec. But maybe the p2p layer is an implementation detail that should just be in the spec